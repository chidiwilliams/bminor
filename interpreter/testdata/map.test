m: map string integer = { "hello" : 5, "goodbye" : 10 };
print m["hello"], "\n";
m["farewell"] = 20;
print m["farewell"], m["goodbye"];
