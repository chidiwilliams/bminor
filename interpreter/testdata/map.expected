5
{ hello: 5, goodbye: 10, farewell: 20 }
